posts/2021-09-29-jcon-conference.adoc

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: "MicroProfile, Jkarta EE and Open Liberty at JCON-Online 2021"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/GraceJansen
+author_github: https://github.com/GraceJansen
+seo-title: MicroProfile, Jkarta EE and Open Liberty at JCON-Online 2021 - OpenLiberty.io
+seo-description: A helpful summary of the JCON-Online 2021 workshops and talks on MicroProfile, Jakarta EE and Open Liberty.
+blog_description: "A helpful summary of the JCON-Online 2021 workshops and talks on MicroProfile, Jakarta EE and Open Liberty."
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= MicroProfile, Jakarta EE and Open Liberty at JCON-Online 2021
+Grace Jansen <https://github.com/GraceJansen>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+ Despite the global pandemic and general chaos of this year, many amazing developer events and conferences are still taking place virtually, bringing the developer community together once more. One such event that will use a digital format to bring together the international Java community is JCON-Online. The great part is, it's free for all JUG (Java User Group) members!
```

Comment:
need to move the text here to left margin- the indent causes it to render as a code block